utils/helper.sh

```diff
@@ -30,13 +30,30 @@ function check_env() {
   return 0
 }
 
+function get_version_confluent_cli() {
+  confluent version | grep "^Version:" | cut -d':' -f2 | cut -d'v' -f2
+}
+
+function version_gt() {
+  test "$(printf '%s\n' "$@" | sort -V | head -n 1)" != "$1";
+}
+
 function validate_version_confluent_cli_v2() {
 
   if [[ -z $(confluent version | grep "Go") ]]; then
     echo "This demo requires the new Confluent CLI. Please update your version and try again."
     exit 1
   fi
 
+  REQUIRED_CONFLUENT_CLI_VER=${1:-"0.265.0"}
+  CONFLUENT_CLI_VER=$(get_version_confluent_cli)
+
+  if version_gt $REQUIRED_CONFLUENT_CLI_VER $CONFLUENT_CLI_VER; then
+    echo "ERROR: demos require Confluent CLI version ${REQUIRED_CONFLUENT_CLI_VER} which is bundled with ${CONFLUENT}. Current reported version: ${CONFLUENT_CLI_VER}"
+    echo -e "Install the required Confluent CLI version with the command ->\n  curl -sL https://cnfl.io/cli | sh -s -- -b $CONFLUENT_HOME/bin v0.265.0"
```

Comment:
@rspurgeon Alternate option: wait for docs to be published with downgrade instructions, and point to docs.  I'm investigating when those will be published.